src/CustomerIOPushMessaging.tsx

```diff
@@ -0,0 +1,33 @@
+import { NativeModules, Platform } from 'react-native';
+
+const LINKING_ERROR =
+  `The package 'customerio-reactnative' doesn't seem to be linked. Make sure: \n\n` +
+  Platform.select({ ios: "- You have run 'pod install'\n", default: '' }) +
+  '- You rebuilt the app after installing the package\n' +
+  '- You are not using Expo managed workflow\n';
+
+/**
+ * Get CustomerIOPushMessaging native module
+ */
+const PushMessagingNative = NativeModules.CustomerioPushMessaging
+  ? NativeModules.CustomerioPushMessaging
+  : new Proxy(
+      {},
+      {
+        get() {
+          throw new Error(LINKING_ERROR);
+        },
+      }
+    );
+
+class CustomerIOPushMessaging {
+  onMessageReceived(message: any): Promise<boolean> {
```

Comment:
From reading [handleMessge](https://github.com/customerio/customerio-reactnative/pull/117/files#diff-74884e5e24c17161d56be01a332a64aeabd1f5bbafefab2bbd409befbb72fdd6R86) function body, the `message` data type is known at compile time so I don't think we should be using `any` here. 

I know that we have more work to do to improve the typescript support for the SDK. I am going to assume that using `any` here is more of a placeholder until better typescript support is added to the SDK. If that assumption is correct, I am OK with using `any` for now.

Reply:
Yes, that's correct. My assumption was same when adding `any` here.